tests/validators/answers/test_number_answer_validator.py

```diff
@@ -175,3 +175,39 @@ def test_invalid_numeric_answers():
     ]
 
     assert validator.errors == expected_errors
+
+
+def test_invalid_maximum_minimum_value_from_answer_source():
+    answer = {
+        "id": "min-max-range",
+        "mandatory": False,
+        "minimum": {"value": {"identifier": "set-minimum", "source": "answers"}},
+        "maximum": {"value": {"identifier": "set-maximum", "source": "answers"}},
+        "type": "Number",
+    }
+
+    answers = {
+        "answers": [
+            answer,
+            {
+                "id": "set-minimum",
+                "type": "Number",
+                "minimum": {"value": -999_999_999_999_999},
```

Comment:
Now that you're covering the test in this PR instead of the other, this value will need updating like it was in the other

Reply:
Updated to `9_999_999_999_999_999` 👍